spotify_confidence/analysis/confidence_utils.py

```diff
@@ -104,170 +112,23 @@ def validate_levels(df: DataFrame, level_columns: Union[str, Iterable], levels:
             )
 
 
-def add_mde_columns(df: DataFrame, mde_column: str) -> DataFrame:
```

Comment:
I created a much cleaner and performant way to add NIMs and MDEs to GenericComputer